portality/background.py

```diff
@@ -59,6 +65,8 @@ def execute(self, background_task):
 
         try:
             background_task.run()
+            if job.outcome_status == BgjobOutcomeStatus.Pending.value:
+                job.outcome_status = BgjobOutcomeStatus.Success
```

Comment:
should we use 'value' attribute for Success?

Reply:
setter of outcome_status will auto call `.value` if input value is enum
```
    @outcome_status.setter
    def outcome_status(self, outcome_status: Union[str, BgjobOutcomeStatus]):
        if isinstance(outcome_status, BgjobOutcomeStatus):
            outcome_status = outcome_status.value
        self._set_with_struct('outcome_status', outcome_status)
```